core-features/src/systemAlert.py

```diff
@@ -0,0 +1,35 @@
+'''
```

Comment:
You can remove this comment section. We can access the API's description via your issue.